verifier.py

```diff
@@ -355,31 +355,68 @@ def __init__(self):
-            'body h2'                                    # Activity points, project count
+            '#projects li',  # Specific project list item
+            'body div.panel-body',  # Component list
+            'body h2'  # Activity points, project count
         ]
         self.harvest_pages(run_info['user_profile_page_urls'], '', UserProfilePage)
         self.size_comparison()
         self.spot_check()
 
 
-class InstitutionProfileVerifier(Verifier):
+class InstitutionDashboardVerifier(Verifier):
     def __init__(self):
         Verifier.__init__(self)
         self.pages = []
         self.minimum_size = 350
         self.page_elements = [
             '#fileBrowser > div.db-infobar > div > div',  # Project preview
-            '#tb-tbody'                                   # Project browser
+            '#tb-tbody'  # Project browser
         ]
         self.harvest_pages(run_info['institution_urls'], '', InstitutionProfilePage)
         self.size_comparison()
         self.spot_check()
 
 
-# Main execution
-
-RegistrationFiles = RegistrationFilesVerifier()
-
-print("Retry list: ", RegistrationFiles.failed_pages)
+def main():
+    # for modularization and then concatenate lists
+    rescrape_list = []
+    for i in range(NUM_RETRIES):
+        if run_info['scrape_nodes']:
+            if run_info['include_files']:
+                project_files = ProjectFilesVerifier()
+                rescrape_list.append(project_files.failed_pages)
+            if run_info['include_wiki']:
+                project_wiki = ProjectWikiVerifier()
+                rescrape_list.append(project_wiki.failed_pages)
+            if run_info['include_analytics']:
+                project_analytics = ProjectAnalyticsVerifier()
+                rescrape_list.append(project_analytics.failed_pages)
+            if run_info['include_registrations']:
+                project_registrations = ProjectRegistrationsVerifier()
+                rescrape_list.append(project_registrations.failed_pages)
+            if run_info['include_forks']:
+                project_forks = ProjectForksVerifier()
+                rescrape_list.append(project_forks.failed_pages)
+            if run_info['include_dashboard']:  # This must go last because its URLs don't have a specific ending.
+                project_dashboards = ProjectDashboardVerifier()
+                rescrape_list.append(project_dashboards.failed_pages)
+        if run_info['scrape_registrations']:
+            # Must run all page types automatically
+            registration_files = RegistrationFilesVerifier()
+            registration_wiki = RegistrationWikiVerifier()
+            registration_analytics = RegistrationAnalyticsVerifier()
+            registration_forks = RegistrationForksVerifier()
+            registration_dashboards = RegistrationDashboardVerifier()
+            rescrape_list.extend((registration_files.failed_pages, registration_wiki.failed_pages,
```

Comment:
`.extend((l1, l2))` seems to append the other lists instead of concatenation. Try:
`l1 = [1,2,3]
l2 = [4,5]
l3 = [6,7]

l3.extend((l1, l2))
print(l3)`

Meanwhile:
`l1 = [1,2,3]
l2 = [4,5]
l3 = [6,7]

l3.extend(l1 + l2)
print(l3)`

And finally:
`l1 = [1,2,3]
l2 = [4,5]
l3 = [6,7]

l3 += l1 + l2
print(l3)
`